docs/Extensibility/token-transformations.md

```diff
@@ -23,7 +23,7 @@ appRunner.Configure(c => c.UseTokenTransformation(
     transformation: MyFunc));
 ```
 
-* __name__ is used for logging and the [Parse Directive](../../../directives)
```

Comment:
this has changed.  you should merge master. i moved Debug and Parse into their own pages.

Reply:
Did a rebase from master 👍

Reply:
@drewburlingame Would you take another look at this. I did a rebase from master. Just want to make sure I did not screw anything up..